src/jdk.jshell/share/classes/jdk/jshell/tool/JavaShellToolBuilder.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2016, 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
(Assuming the header is intended:) there must be a comma after the year (i.e. `2024,` instead of `2024`).

Reply:
Thanks - fixed in 1769df6.